megablocks/layers/arguments.py

```diff
@@ -38,8 +38,8 @@ class Arguments:
 
     # Compute arguments.
     memory_optimized_mlp : bool = False
-    mlp_type: str = 'mlp'
-    grouped_mlp: bool = False
+    mlp_type : str = 'mlp'
```

Comment:
Did you mean to add these spaces? Looks like we're actually mixed on having them and not having them in this file...

Reply:
hmmm yea I thought that the spaces would match the existing style of the file!